build/stf-run-ci/README.md

```diff
@@ -12,6 +12,62 @@ Requirements
 - Ansible
 - `oc` command line tool
 
+Variables
+---------
+
+Not all variables are listed here, but these are the most common ones you might
+choose to override:
+
+| Parameter name                                  | Values       | Default   | Description                                                                                           |
+| ------------------------------                  | ------------ | --------- | ------------------------------------                                                                  |
+| `__deploy_stf`                                  | {true,false} | true      | Whether to deploy an instance of STF                                                                  |
+| `__local_build_enabled`                         | {true,false} | true      | Whether to deploySTF from local built artifacts. Also see `working_branch`, `sg_branch`, `sgo_branch` |
+| `sg_branch`                                     | <git_branch> | master    | Which Smart Gateway git branch to checkout                                                            |
+| `sgo_branch`                                    | <git_branch> | master    | Which Smart Gateway git branch to checkout                                                            |
```

Comment:
Which Smart Gateway **Operator** git branch to checkout

Reply:
Got it, thanks!